src/software/ai/hl/stp/play/kickoff_enemy_play.cpp

```diff
@@ -82,22 +82,43 @@ void KickoffEnemyPlay::getNextTactics(TacticCoroutine::push_type &yield,
         std::make_shared<MoveTactic>(true), std::make_shared<MoveTactic>(true),
         std::make_shared<MoveTactic>(true)};
 
+    // created a new local variable Point at the center of the field (0,0)
+    Point point = world.field().centerPoint();
```

Comment:
nit: you can probably just use the point, no need to make a local copy here

Reply:
done